recipe/meta.yaml

```diff
@@ -10,7 +10,8 @@ source:
   sha256: eb2e116e75ecef9d4d228fdc66af54269afa26ab4463042e33785b887c628ba8
 
 build:
-  number: 0
+  number: 1
+  noarch: python
   script: python setup.py install
```

Comment:
While we're at it, let's switch this to `pip install --no-deps .` , which we're using for most Jupyter repos. The advantage of that is that pip 'sees' the package when it's installed by conda, so it's less likely to try to install it again and cause problems.